.github/workflows/terraform-databricks.yml

```diff
@@ -0,0 +1,191 @@
+# This is a basic GitHub Actions pipeline to validate Terraform
+
+name: "Databricks Terraform Infrastructure Pipeline"
+
+on:
+  workflow_call:
+    inputs:
+      DATABRICKS_HOST:
+        required: true
+        type: string
+      DATABRICKS_ACCOUNT_ID:
+        required: true
+        type: string
+      ARM_CLIENT_ID:
+        required: true
+        type: string
+      ARM_SUBSCRIPTION_ID:
+        required: true
+        type: string
+      ARM_TENANT_ID:
+        required: true
+        type: string
+      BACKEND_RG_NAME:
+        required: true
+        type: string
+      BACKEND_SA_NAME:
+        required: true
+        type: string
+      BACKEND_CONTAINER_NAME:
+        required: true
+        type: string
+      BACKEND_KEY:
+        required: true
+        type: string
+      TERRAFORM_VERSION:
+        required: true
+        type: string
+    secrets:
+      ARM_CLIENT_SECRET:
+        required: true
+
+defaults:
+  run:
+    working-directory: terraform
+
+env:
+  DATABRICKS_HOST: ${{ inputs.DATABRICKS_HOST }}
+  DATABRICKS_ACCOUNT_ID: ${{ inputs.DATABRICKS_ACCOUNT_ID }}
+  ARM_CLIENT_ID: ${{ inputs.ARM_CLIENT_ID }}
+  ARM_SUBSCRIPTION_ID: ${{ inputs.ARM_SUBSCRIPTION_ID }}
+  ARM_TENANT_ID: ${{ inputs.ARM_TENANT_ID }}
+  BACKEND_RG_NAME: ${{ inputs.BACKEND_RG_NAME }}
+  BACKEND_SA_NAME: ${{ inputs.BACKEND_SA_NAME }}
+  BACKEND_CONTAINER_NAME: ${{ inputs.BACKEND_CONTAINER_NAME }}
+  BACKEND_KEY: ${{ inputs.BACKEND_KEY }}
+  ARM_CLIENT_SECRET: ${{ secrets.ARM_CLIENT_SECRET }}
+  TERRAFORM_VERSION: ${{ inputs.TERRAFORM_VERSION }}
+
+jobs:
+  build:
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Terraform
+        uses: hashicorp/setup-terraform@v2.0.0
+        with:
+          terraform_version: $TERRAFORM_VERSION
+
+      #      - name: Set up Databricks CLI profile for Terraform provider
+      #        run: |
+      #          echo "[DEFAULT]" >> ~/.databrickscfg
+      #          echo "host = $DATABRICKS_HOST" >> ~/.databrickscfg
+      #          echo "token = $DATABRICKS_TOKEN" >> ~/.databrickscfg
+
+      - name: Check formatting of Terraform code
```

Comment:
wydaje mi sie ze to nic nie robi (?)

Reply:
no taki oficjalny linter, mozna dodac tfsec i tflint